x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/__mocks__/rule_type.ts

```diff
@@ -76,6 +76,7 @@ export const createRuleTypeMocks = (
     scopedClusterClient: elasticsearchServiceMock.createScopedClusterClient(),
     alertFactory: {
       create: jest.fn(() => ({ scheduleActions })),
+      hasReachedAlertLimit: () => false,
```

Comment:
**question:** is it possible to convert the code within security solution to use the mocked alert factory from `x-pack/plugins/alerting/server/mocks.ts`?